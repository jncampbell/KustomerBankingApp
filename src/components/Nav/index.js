import React from 'react';
require('./styles/nav.scss');

const Nav = () => (
  <nav className='site-nav'>
    <div className='site-nav__logo'><img src='./assets/money-bag.png' /></div>
  </nav>
);

export default Nav;
