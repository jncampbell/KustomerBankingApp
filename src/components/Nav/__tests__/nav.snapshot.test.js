import React from 'react';
import renderer from 'react-test-renderer';
import Nav from '../index';

describe('Nav', () => {
  it('has consistent render output', () => {
    const component = renderer.create(
      <Nav />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
