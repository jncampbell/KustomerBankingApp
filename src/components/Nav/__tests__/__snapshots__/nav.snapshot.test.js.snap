// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav has consistent render output 1`] = `
<nav
  className="site-nav"
>
  <div
    className="site-nav__logo"
  >
    <img
      src="./assets/money-bag.png"
    />
  </div>
</nav>
`;
